relays/substrate-client/src/chain.rs

```diff
@@ -19,16 +19,22 @@ use crate::client::Client;
 use bp_runtime::Chain as ChainBase;
 use frame_support::Parameter;
 use jsonrpsee::common::{DeserializeOwned, Serialize};
-use sp_core::Pair;
+use num_traits::{CheckedSub, Zero};
+use sp_core::{storage::StorageKey, Pair};
 use sp_runtime::{
 	generic::SignedBlock,
 	traits::{AtLeast32Bit, Dispatchable, MaybeDisplay, MaybeSerialize, MaybeSerializeDeserialize, Member},
 	Justification,
 };
-use sp_std::fmt::Debug;
+use std::{fmt::Debug, time::Duration};
 
 /// Substrate-based chain from minimal relay-client point of view.
 pub trait Chain: ChainBase {
+	/// Chain name.
+	const NAME: &'static str;
+	/// Average block interval.
```

Comment:
Can you expand on the doc here? Is it meant to be equal to block time on the actual chain, or is it some other polling scheme?

Reply:
I could change to "Average interval between blocks" :) Not sure how to describe it better. No matter how it would be used (either to poll, or to do any computations), it should be near to average time between blocks on real chain.

Reply:
```suggestion
	/// Average block interval.
	/// 
	/// How often blocks are produced on that chain. It's suggested to set this value to match the block time of the chain.
```